final/docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: "3.8"
+services:
+  bitcoin-core:
+    build:
+      dockerfile: ./bitcoind/Dockerfile
+    image: bitcoind
+    container_name: bitcoind
+    command:
+      - --testnet
+      - -conf=/home/bitcoin/.bitcoin/bitcoin.conf
```

Comment:
these two lines could go in the entrypoint.sh startup script

Reply:
i think we can keep `testnet` here because if someone wishes to try `regtest` they can just change in `docker-compose` and use

Reply:
I think we should keep `conf` here as well so that both the `network` and its corresponding configuration can be given from the `docker-compose` itself.